src/components/button/Button.tsx

```diff
@@ -92,6 +95,7 @@ class Button extends React.Component<ButtonProps> {
         let button = (
             // eslint-disable-next-line react/button-has-type
             <button
+                aria-label={ariaLabel}
```

Comment:
the Button component uses the `rest` functionality to allow additional HTML attributes. this is how other components have been adding aria labels:
https://github.com/box/box-ui-elements/blob/d249a01de01dee59b8f53e64e1b569ddbe4f7c4f/src/components/button/Button.tsx#L75
https://github.com/box/box-ui-elements/blob/d249a01de01dee59b8f53e64e1b569ddbe4f7c4f/src/elements/common/sub-header/SortButton.js#L28
https://github.com/box/box-ui-elements/blob/d249a01de01dee59b8f53e64e1b569ddbe4f7c4f/src/elements/content-sidebar/SidebarNavSignButton.tsx#L59

are we able to continue using this pattern? any specific reason for the explicit prop now?

Reply:
Thank you for showing me examples. I made changes to use the `rest` functionality.